src/core/generators/schemaDefinition.ts

```diff
@@ -71,22 +71,25 @@ export const generateSchemasDefinition = async (
           );
 
           if (!imp) {
-            output += `export type ${schemaName} = ${resolvedValue.value};\n`;
+            const outputValue = resolvedValue.value === '{}' ? 'any' : resolvedValue.value;
+            output += `export type ${schemaName} = ${outputValue};\n`;
           } else {
-            const alias = imp?.specKey
+            let alias = imp?.specKey
               ? `${pascal(getSpecName(imp.specKey, context.specKey))}${
                   resolvedValue.value
                 }`
               : `${resolvedValue.value}Bis`;
 
+            alias = alias === '{}' ? 'any' : alias;
```

Comment:
will never be `{}` in that case

Reply:
I don't think you need to change the alias here, just the output value should be changed or am I wrong?

Reply:
In that case, that will be always a schema so it's always a name